templates/zerver/help/add-or-remove-users-from-a-stream.md

```diff
@@ -17,13 +17,20 @@ to a stream][configure-invites].
 
 1. Select a stream.
 
+1. Select the **Subscribers** tab on the right.
+
```

Comment:
I wonder if we should factor out everything through selecting the subscribers tab into a
`{relative|stream|all|subscribers}` macro we can just include?

Reply:
Hmm... we don't know what stream they want to select. So this would be changing the whatever causes the tab to be loaded to a particular tab when you click on a stream. Which seems to default to 'Personal' if it's your first time in the session accessing the view and "General' if you're not subscribed to the stream. And if you're returning to the view in a session, it loads the last tab that was loaded.

Maybe a note for looking into?